dcmgr/lib/dcmgr/node_modules/vnet_collector.rb

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+require 'isono'
+require 'ipaddress'
+
+require 'vnet_api_client'
+
+module Dcmgr
+  module NodeModules
+    class VnetCollector < Isono::NodeModules::Base
+      include Isono::NodeModules
+      include Dcmgr::Logger
+
+      initialize_hook do
+        endpoint = Dcmgr::Configurations.dcmgr.features.vnet_endpoint
+        port = Dcmgr::Configurations.dcmgr.features.vnet_endpoint_port
```

Comment:
The default values are defined here.

https://github.com/axsh/wakame-vdc/blob/develop/dcmgr/lib/dcmgr/configurations/features.rb#L25-L29

for `bin/collector` I will fix it.